pkg/cloud/scope/cluster.go

```diff
@@ -98,6 +100,23 @@ type ClusterScope struct {
 	controllerName  string
 }
 
+func (s *ClusterScope) KubeadmControlPlaneBindPort() int32 {
```

Comment:
Missing description of what this method does.

Reply:
Done